tool/scrape_mdn.dart

```diff
@@ -166,6 +173,24 @@ String convertMdnToMarkdown(String content) {
     lines.removeLast();
   }
 
+  // Rewrite relative link references:
+  //   "[WebGL API](/en-US/docs/Web/API/WebGL_API)"
+  final linkRefRegex = RegExp(r'\[([^\]]+)\]\(([\w\/-]+)\)');
+  // ignore: prefer_expression_function_bodies
```

Comment:
I'm guessing the formatter does some weird wrapping if we make this an expression instead? Maybe add a comment on why we ignore the lint.

Reply:
It's not terrible (the formatting) - I just tend not to use the shorthand syntax for functions that are longer than a line or two:

<img width="623" alt="Screenshot 2024-03-12 at 12 38 29 PM" src="https://github.com/dart-lang/web/assets/1269969/ef436a45-aec8-4f90-9d5f-ae793be41917">

Reply:
(removed the todo and re-formatted the expression)

Reply:
Yeah I also tend to avoid the expression syntax for long functions. Either way sounds fine to me, thanks!